src/runner.js

```diff
@@ -149,7 +156,7 @@ exports.run = function(config) {
       if (tunnelHost) {
         config.states = transformToTunnelHost(config.states, config.tunnel.host, tunnelHost);
       }
-      var payload = omit(config, ['apiKey', 'resolution', 'resolutions', 'includeRules', 'excludeRules', 'tunnel', 'failureExitCode', 'sauce.launchSauceConnect']);
+      var payload = omit(config, ['apiKey', 'username', 'accessKey', 'resolution', 'resolutions', 'includeRules', 'excludeRules', 'tunnel', 'failureExitCode', 'sauce.launchSauceConnect']);
```

Comment:
I think we may need to keep this. Currently the `sauce` option gets passed to test-runner eventually with sauce creds. Or we keep this the way it is, and let's have a separate ticket to think through how to best handle that scenario

Reply:
Sorry can you clarify this a bit? Thanks @loyalchow